src/helm/benchmark/static/schema.yaml

```diff
@@ -23,6 +23,13 @@ models:
     access: limited
     num_parameters: 17000000000
     release_date: 2022-05-03
+  - name: ai21/j1-grande-v2-beta
```

Comment:
Just curious: is this endpoint stable given that it's beta?

Reply:
I followed up on this. Will double check before merging.